lib/__tests__/cli.test.mjs

```diff
@@ -36,7 +36,6 @@ describe('buildCLI', () => {
 			color: false,
 			disableDefaultIgnores: false,
 			fix: false,
-			formatter: 'string',
```

Comment:
Which tests are failing when you leave it there?
i.e. I suppose both

https://github.com/stylelint/stylelint/blob/9bc35126c63677954dbb28aeecc6efee9cda0325/lib/__tests__/cli.test.mjs#L91-L93

and

https://github.com/stylelint/stylelint/blob/9bc35126c63677954dbb28aeecc6efee9cda0325/lib/__tests__/cli.test.mjs#L108-L111

were expected to overwrite it.

Reply:
Only `flags - default` fail since the default is no longer only a flag, but in separate function (ie: `getFormatter`).